kani-driver/src/call_single_file.rs

```diff
@@ -93,19 +85,61 @@ impl KaniSession {
             flags.push("--enable-stubbing".into());
         }
         if let Some(harness) = &self.args.harness {
-            flags.push(format!("--harness={harness}").into());
+            flags.push(format!("--harness={harness}"));
         }
 
+        // This argument will select the Kani flavour of the compiler. It will be removed before
+        // rustc driver is invoked.
+        flags.push("--goto-c".into());
+
         #[cfg(feature = "unsound_experiments")]
         flags.extend(self.args.unsound_experiments.process_args());
 
         flags
     }
 
-    /// These arguments are arguments passed to kani-compiler that are `rustc` specific.
-    /// These are also used by call_cargo to pass as the env var KANIFLAGS.
+    /// This function generates all rustc configurations required by our goto-c codegen.
+    ///
+    /// We override the `std` library using the mechanism described here:
+    /// <https://rust-lang.zulipchat.com/#narrow/stream/182449-t-compiler.2Fhelp/topic/.E2.9C.94.20Globally.20override.20an.20std.20macro/near/268873354>
+    /// for more details.
```

Comment:
Remove this line.

Reply:
Sorry, I just meant the last line of the comment: 
```
/// for more details.
```
to fix the sentence and not the entire comment. The rest of the comment is still relevant.